### PR TITLE
style : cards added

### PR DESCRIPTION
**Pull request by vaibhav18matere**
horizontal, vertical, badge, dismiss, text-overlay, text-only cards added

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -37,14 +37,17 @@ html, body{
...
   --danger:#842029;
-  --info:#111212;
+  --cart-btn:#fa0404;
+  --card-border:#dad9d9;
 }
 .mb{
   margin-block-start: 0;
   margin-block-end: 0;
 }
+.mt3{
```
Comment (Raghav888) [inline styles.css]: Please use better naming convention

**Review by Raghav888 — APPROVED** [review]
Approved

File: components/card/card.html [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+                    <div class="dismiss-card__description">
+                        <h3>Ashneer Grover - Bharat Pay</h3>
+                        <small>Bhai, tujhse to ye nahi ho payega. band kar ye sab...</small>
+                        <strong>Ye sab doglapan hai</strong>
+                        <button class="btn btn-primary">Dhanda Banao</button>
+                    </div>
+                </div>
+            </div>
+            <div class="dismiss-card-carbon-container">
+                <!-- <iframe
```
Comment (Jia-A) [inline components/card/card.html]: Remove the commented code.

**Review by Jia-A — COMMENTED** [review]
Looks good to me. Maybe just add a screenshot or something for visualization.

File: components/card/card.html (hunk 2 above)
Comment (vaibhav18matere) [reply components/card/card.html]: thanks !

File: styles.css (hunk 1 above)
Comment (vaibhav18matere) [reply styles.css]: sure, thanks!

**Pull request merged**